pkg/api/roles.go

```diff
@@ -97,8 +99,9 @@ func (hs *HTTPServer) declareFixedRoles() error {
 		{
 			Role: accesscontrol.RoleDTO{
 				Version:     2,
-				Name:        "fixed:datasources:id:viewer",
-				Description: "Gives access to read datasources ID",
+				Name:        "fixed:datasources:id:reader",
+				DisplayName: "Datasource id reader",
```

Comment:
nice! this reminds me to push through the displayname update to be sentences cased

Reply:
```suggestion
				DisplayName: "Data source ID reader",
```